staging/src/k8s.io/client-go/tools/leaderelection/leaderelection.go

```diff
@@ -158,11 +162,14 @@ func (le *LeaderElector) Run(ctx context.Context) {
 
 // RunOrDie starts a client with the provided config or panics if the config
 // fails to validate.
-func RunOrDie(ctx context.Context, lec LeaderElectionConfig) {
+func RunOrDie(ctx context.Context, lec LeaderElectionConfig, lc LeaderChecker) {
```

Comment:
This is a breaking change to the client library. Can we add it to the Config object instead?